go/capture/driver/airbyte/driver.go

```diff
@@ -281,26 +285,45 @@ func (d driver) Validate(ctx context.Context, req *pc.ValidateRequest) (*pc.Vali
 	return resp, nil
 }
 
-func (d driver) Capture(req *pc.CaptureRequest, stream pc.Driver_CaptureServer) error {
+// ApplyUpsert is a no-op (not supported by Airbyte connectors).
+func (d driver) ApplyUpsert(context.Context, *pc.ApplyRequest) (*pc.ApplyResponse, error) {
```

Comment:
Because we're going through the airbyte spec, these are effectively no-ops.

But they're needed (and it has me considering whether we ought to make the capture protocol the native protocol for our own connectors). A good example of why is replication slots in CDC databases, which are a created entity that consumes DB resources. We want to create slots when the capture is created, and delete it when the capture goes away. But today we can't thread this through w/ the airbyte protocol.

Reply:
I see. Cool!